packages/ckeditor5-link/src/linkconfig.ts

```diff
@@ -9,6 +9,44 @@
 
 import type { ArrayOrItem } from 'ckeditor5/src/utils.js';
 
+/**
+ * This is a list of most commonly used protocols to make the link custom configuration a bit easier.
+ * The {@link module:link/linkconfig~LinkConfig#allowedProtocols `allowedProtocols` } list
+ * can also contain other values than enlisted here.
+ */
+export enum COMMON_PROTOCOLS {
```

Comment:
I am not sure if we need this enum. Who will use it and why? 
I'm pretty sure that if someone wants to use custom protocols, they exactly know which one.

Reply:
I think this is not needed and people will not use it. They will go with `allowedProtocols: [ 'http', 'ftp', 'foobar' ]`.

Reply:
okay, a proposition as said ;) will have only literals